releases/release-1.31/release-notes/release-notes-draft.json

```diff
@@ -646,7 +759,7 @@
   "123102": {
     "commit": "a7832cc1698ed1e40283840280b1fd697f4f495e",
     "text": "kubeadm: during the validation of existing kubeconfig files on disk, handle cases where the \"ca.crt\" is a bundle and has intermediate certificates. Find a common trust anchor between the \"ca.crt\" bundle and the CA in the existing kubeconfig on disk instead of treating \"ca.crt\" as a file containing a single CA.",
-    "markdown": "Kubeadm: during the validation of existing kubeconfig files on disk, handle cases where the \"ca.crt\" is a bundle and has intermediate certificates. Find a common trust anchor between the \"ca.crt\" bundle and the CA in the existing kubeconfig on disk instead of treating \"ca.crt\" as a file containing a single CA. ([#123102](https://github.com/kubernetes/kubernetes/pull/123102), [@astundzia](https://github.com/astundzia)) [SIG Cluster Lifecycle]",
+    "markdown": "Kubeadm: during the validation of existing kubeconfig files on disk, handle cases where the \"ca.crt\" is a bundle and has intermediate certificates. Find a common trust anchor between the \"ca.crt\" bundle and the CA in the existing kubeconfig on disk instead of treating \"ca.crt\" as a file containing a single CA. ([#123102](https://github.com/kubernetes/kubernetes/pull/123102), [@astundzia](https://github.com/astundzia))",
```

Comment:
```suggestion
    "markdown": "Kubeadm: during the validation of existing kubeconfig files on disk, handle cases where the \"ca.crt\" is a bundle and has intermediate certificates. Find a common trust anchor between the \"ca.crt\" bundle and the CA in the existing kubeconfig on disk instead of treating \"ca.crt\" as a file containing a single CA. ([#123102](https://github.com/kubernetes/kubernetes/pull/123102), [@astundzia](https://github.com/astundzia)) [SIG Cluster Lifecycle]",
```